bank/persistence/account_operations_test.go

```diff
@@ -0,0 +1,103 @@
+package persistence_test
+
+import (
+	"github.com/iliyanmotovski/bankv1/bank/domain"
+	"github.com/iliyanmotovski/bankv1/bank/persistence"
+	"github.com/iliyanmotovski/bankv1/bank/persistence/db_config"
+	"gopkg.in/mgo.v2/bson"
+	"testing"
+)
+
+func TestItWritesAndReadsAccountData(t *testing.T) {
+	db := db_config.NewDatabase()
```

Comment:
name of the package looks misleading. 

testdb.New() 
databasetest.New() 
or something sounds better.